skeleton/frontend/src-bin/main.hs

```diff
@@ -1,10 +1,6 @@
 import Frontend
 import Common.Route
 import Obelisk.Frontend
-import Obelisk.Route.Frontend
-import Reflex.Dom
 
 main :: IO ()
-main = do
-  let Right validFullEncoder = checkEncoder fullRouteEncoder
-  run $ runFrontend validFullEncoder frontend
+main = runFrontend fullRouteEncoder frontend
```

Comment:
Can't this be something like `main = Frontend.main` instead ?

Reply:
Point, that would be maximally future-proof.

Reply:
Okay, did this for the frontend, as well as the backend to make it symmetric.